api/views.py

```diff
@@ -108,13 +108,23 @@ def phone_list(self, request, format=None):
 
         # collect list of all users that have door access
         users_with_door_access = []
+
+        # Members with active door access
         for ss in (
             ServiceSubscription.objects.select_related("user")
             .filter(service=drfx_settings.DEFAULT_ACCOUNT_SERVICE)
             .filter(state=ServiceSubscription.ACTIVE)
         ):
             users_with_door_access.append(ss.user)
 
+        # Members with overdue door access
+        for ss in (
+            ServiceSubscription.objects.select_related("user")
+            .filter(service=drfx_settings.DEFAULT_ACCOUNT_SERVICE)
+            .filter(state=ServiceSubscription.OVERDUE)
+        ):
+            users_with_door_access.append(ss.user)
+
```

Comment:
Add this import

```
from django.db.models import Q
```

and

```suggestion

        # Members with active door access
        for ss in (
            ServiceSubscription.objects.select_related("user")
            .filter(service=drfx_settings.DEFAULT_ACCOUNT_SERVICE)
            .filter(Q(state=ServiceSubscription.ACTIVE) | Q(state=ServiceSubscription.OVERDUE))
        ):
            users_with_door_access.append(ss.user)
```

More info about Q here: https://docs.djangoproject.com/en/4.1/topics/db/queries/#complex-lookups-with-q-objects

(untested)